Description: Removes test "executes rpc instructions"
 This test fails reliably on arm64 and misp64el.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2019-11-04

--- elixir-lang-1.9.1.dfsg.orig/lib/mix/test/mix/tasks/release_test.exs
+++ elixir-lang-1.9.1.dfsg/lib/mix/test/mix/tasks/release_test.exs
@@ -268,27 +268,6 @@ defmodule Mix.Tasks.ReleaseTest do
     end)
   end
 
-  @tag :epmd
-  test "executes rpc instructions" do
-    in_fixture("release_test", fn ->
-      config = [releases: [permanent1: [include_erts: false]]]
-
-      Mix.Project.in_project(:release_test, ".", config, fn _ ->
-        root = Path.absname("_build/dev/rel/permanent1")
-        Mix.Task.run("release")
-        script = Path.join(root, "bin/permanent1")
-
-        open_port(script, ['start'])
-        wait_until_decoded(Path.join(root, "RELEASE_BOOTED"))
-        assert System.cmd(script, ["rpc", "ReleaseTest.hello_world"]) == {"hello world\n", 0}
-        assert System.cmd(script, ["stop"]) == {"", 0}
-
-        assert {pid, 0} = System.cmd(script, ["pid"])
-        assert pid != "\n"
-      end)
-    end)
-  end
-
   test "runs eval and version commands" do
     # In some Windows setups (mostly with Docker), `System.cmd/3` fails because
     # the path to the command/executable and one or more arguments contain spaces.
